fastlane/swift/Runner.swift

```diff
@@ -95,6 +137,7 @@ extension Runner {
         }
         
         socketClient.connectAndOpenStreams()
+        print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
        verbose(message: "Leave \(#function) \(#line)")
```